mycroft/skills/__init__.py

```diff
@@ -11,3 +11,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
if we are going to refactor, should we consider moving code out of __init__.py?

Reply:
This `__init__.py` doesn't contain any code? Any more would definitely be outside of the scope of this PR since it focus on splitting core.py into logical pieces preparing for futurue refactoring on the MycroftSkill class.